### PR TITLE
Examples update for Dataset-JSON v1.1

### PR DESCRIPTION
**Pull request by lexjansen**
Updates for ADaM, SDTM, and SEND.

- ADaM examples implement sending numeric dates as ISO 8601 strings (with displayFormat E8601DA.)
- Also, ADaM examples implement decimal transfer according to the following logic (just as an example!!)
`  
if xml_datatype in ('float' 'integer') and
     dataset_name in ('ADADAS' 'ADLBC' 'ADLBCPV' 'ADLBH' 'ADLBHPV' 'ADLBHY' 'ADNPIX' 'ADSL' 'ADVS') and
     name in ('PCHG' 'AVAL' 'BASE' 'CHG' 'PCHG' 'R2A1LO' 'R2A1HI' 'BR2A1LO' 'BR2A1HI' 'A1LO' 'A1HI' 'R2A1LO' 'R2A1HI' 'ALBTRVAL' 'LBSTRESN' 'HEIGHTBL' 'WEIGHTBL')
     then do;
      dataType='decimal';
      targetDataType='decimal';
    end;
`
- SDTM and SEND examples have decimal transfers, mostly for --STRESN

### CODE REVIEW

File: examples/adam/adadas.json
Comment (gerrycampion) [inline examples/adam/adadas.json:1]: Are you going to update the ndjson files?

File: examples/sdtm/ae.json
Comment (gerrycampion) [inline examples/sdtm/ae.json:1]: General question - I see, for example, that dataset-json floats and ints are stored as json numbers and dataset-json decimals are stored as json strings. Are these mappings documented somewhere or in the schema?

**Review by gerrycampion — CHANGES_REQUESTED** [review]

File: examples/sdtm/ae.json
Comment (lexjansen) [reply examples/sdtm/ae.json:1]: See the specification: https://wiki.cdisc.org/display/DSJSON1DOT1/Dataset-JSON+1.1. To preserve precision there is an option to transfer fractional numbers as decimals. The specification does not describe when that option should be used.

File: examples/adam/adadas.json
Comment (lexjansen) [reply examples/adam/adadas.json:1]: Sam is updating the ndjson files.

**Comment by lexjansen** [comment]
The sourceSystem name and version properties in the examples are not right. I will update them.

**Comment by swhume** [comment]
I converted the previously committed examples to NDJSON and validated them. They looked good. I will re-run the conversion with this latest commit. Should be good to approve by tomorrow.

**Comment by lexjansen** [comment]
Updates for ADaM, SDTM, and SEND.

- ADaM examples implement sending numeric dates as ISO 8601 strings (with displayFormat E8601DA.)
- Also, ADaM examples implement decimal transfer according to the following logic (just as an example!!)
  `  
if dataset_name in ('ADLBC' 'ADLBH') and 
     name in ('PCHG' 'AVAL' 'BASE' 'CHG' 'PCHG' 'A1LO' 'A1HI' 'R2A1LO' 'R2A1HI' 'BR2A1LO' 'BR2A1HI' 'ALBTRVAL' 'LBSTRESN')
     then do;
      dataType='decimal';
      targetDataType='decimal';
 end;
`

**Review by swhume — APPROVED** [review]
Successfully reviewed, validated, and converted the examples.

File: examples/adam/adadas.json
Comment (swhume) [reply examples/adam/adadas.json:1]: I have converted the examples to NDJSON and will create a PR to add them

**Pull request merged**